test/unit/gateways/versa_pay_test.rb

```diff
@@ -53,12 +53,22 @@ def test_build_store_unstore_request_url
   def test_error_code_from_errors
     # a HTTP 412 response structure
     error = @gateway.send(:error_code_from, { 'success' => false, 'errors' => ['fund_address_unspecified'], 'response_code' => 999 }, 'sale')
-    assert_equal error, 'response_code: 999'
+    assert_equal error, nil
```

Comment:
It's weird to have `error_code` as `nil`, but since the `error_code` is a specific gateway error, it makes sense to return it as a GSRF. 👍🏼